### ISSUE TITLE
How to Train with Multiple GPUs

### ISSUE THREAD

**Issue by haoxiangzhaotju** [dvlab-research/LISA#73]
I didn't find the corresponding multi-GPU training code in train_ds.py. 

    model_engine, optimizer, train_loader, scheduler = deepspeed.initialize(
        model=model,
        model_parameters=model.parameters(),
        training_data=train_dataset,
        collate_fn=partial(
            collate_fn,
            tokenizer=tokenizer,
            conv_type=args.conv_type,
            use_mm_start_end=args.use_mm_start_end,
            local_rank=args.local_rank,
        ),
        config=ds_config,
    )

Does using DeepSpeed automatically enable multi-GPU training?

**Comment by X-Lai** [comment]
Yes, you're correct.
